list_of_prime_numbers/main.py

```diff
@@ -1,10 +1,19 @@
-
 def _is_prime(number):
-    pass
-
+    if number == 1:
+      return False
+    for i in range((number - 1), 1, -1):  
+      if number % i == 0:
+         return False
+    return True    
+   
 
```

Comment:
Nice and concise.  Very effective use of the parameters of a for loop. Only possible issue is that 0 will come back as True/a prime number